src/main/java/com/redhat/devtools/intellij/tektoncd/ui/toolwindow/WindowToolFactory.java

```diff
@@ -56,48 +55,39 @@
 public class WindowToolFactory implements ToolWindowFactory {
     @Override
     public void createToolWindowContent(@NotNull Project project, @NotNull ToolWindow toolWindow) {
-        try {
-            ContentFactory contentFactory = ContentFactory.SERVICE.getInstance();
-
-            TektonTreeStructure structure = new TektonTreeStructure(project);
-            StructureTreeModel<TektonTreeStructure> model = TreeHelper.buildModel(structure, project);
-            new MutableTektonModelSynchronizer<>(model, structure, structure);
-            Tree tree = new Tree(new AsyncTreeModel(model, project));
-            tree.putClientProperty(Constants.STRUCTURE_PROPERTY, structure);
-            tree.setCellRenderer(new NodeRenderer());
-            ActionManager actionManager = ActionManager.getInstance();
-            ActionGroup group = (ActionGroup)actionManager.getAction("com.redhat.devtools.intellij.tektoncd.tree");
-            PopupHandler.installPopupHandler(tree, group, ActionPlaces.POPUP, actionManager, new TektonTreePopupMenuListener());
+        ContentFactory contentFactory = ContentFactory.SERVICE.getInstance();
```

Comment:
Those kind of changes should be prevented, they make it hard for external reviewers to read the history

Reply:
This change in order is necessary when the disposable to be used is not a parameter any longer but a disposable that's created in the sequence.

Reply:
OK I did not catch the reformatting was cause because of the try/catch removal